### PR TITLE
Eth proxy acceptance test

### PR DESCRIPTION
**Pull request by jframe**
## PR Description
This adds an acceptance test to the Eth1 RPC proxy feature being ported from EthSigner to Web3Signer.

## Fixed Issue(s)

Part of https://github.com/ConsenSys/web3signer/issues/763

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java [hunk 1]
```diff
@@ -69,6 +72,7 @@ public class Signer extends FilecoinJsonRpcEndpoint {
   private final Vertx vertx;
   private final String urlFormatting;
   private final Optional<ClientTlsConfig> clientTlsConfig;
+  private JsonRpc2_0Web3j signerJsonRpc;
```
Comment (gfukushima) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java:75]: nit: not sure why we named this jsonRPC instance signer if there isn't nothing signer specifically about it

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/tls/support/MockBalanceReporter.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+package tech.pegasys.web3signer.dsl.tls.support;
+
+import tech.pegasys.web3signer.core.service.jsonrpc.JsonRpcRequest;
+import tech.pegasys.web3signer.core.service.jsonrpc.JsonRpcRequestHandler;
+import tech.pegasys.web3signer.core.service.jsonrpc.handlers.HttpResponseFactory;
+
+import io.vertx.ext.web.RoutingContext;
+
+public class MockBalanceReporter implements JsonRpcRequestHandler {
```
Comment (gfukushima) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/tls/support/MockBalanceReporter.java:21]: Think this should be moved out of tls package...to the shared utils maybe?

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/eth1rpc/EthRpcDownstreamTlsAcceptanceTest.java [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        new BasicKeyStoreOptions(presentedCert.getPkcs12File().toPath(), clientPasswordFile);
+    final ClientTlsOptions clientTlsOptions =
+        new BasicClientTlsOptions(keyStoreOptions, Optional.of(fingerPrintFilePath), true);
+    builder.withDownstreamTlsOptions(clientTlsOptions).withDownstreamHttpPort(downstreamWeb3Port);
+
+    startSigner(builder.build());
+  }
+
+  @Test
+  void ethSignerProvidesSpecifiedClientCertificateToDownStreamServer(@TempDir Path workDir)
```
Comment (gfukushima) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/eth1rpc/EthRpcDownstreamTlsAcceptanceTest.java:87]: is the method name still making reference to ethSigner? should we change that?

File: gradle.properties [hunk 4]
```diff
@@ -1,3 +1,9 @@
+besuVersion=22.10.3
```
Comment (gfukushima) [inline gradle.properties:1]: Could we update this?

**Review by gfukushima — CHANGES_REQUESTED** [review]
lgtm a few minor questions and requests

File: gradle.properties (hunk 4 above)
Comment (jframe) [reply gradle.properties:1]: No unfortunately as newer versions use Java 17. And we blocked by the Azure issue to use Java 17 at the moment.

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/eth1rpc/EthRpcDownstreamTlsAcceptanceTest.java (hunk 3 above)
Comment (jframe) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/eth1rpc/EthRpcDownstreamTlsAcceptanceTest.java:87]: Yes definitely, good catch. Replaced all the other ethsigner references as well

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java (hunk 1 above)
Comment (jframe) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/Signer.java:75]: yes not sure either. I think copied some code that used that name. but doesn't make sense. renamed it to jsonRpc

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/tls/support/MockBalanceReporter.java (hunk 2 above)
Comment (jframe) [reply acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/tls/support/MockBalanceReporter.java:21]: done

**Review by gfukushima — APPROVED** [review]
Lgtm

**Pull request merged**